app/controllers/main/routes.py

```diff
@@ -188,19 +186,19 @@ def viewUsersProfile(username):
                                 totalSustainedEngagements = totalSustainedEngagements,
                             )
     abort(403)
-
-@main_bp.route('/profile/<username>/emergencyContact', methods=['GET', 'POST'])
+###############################EMERGENCY CONTACT######################################
```

Comment:
This seemed to be for finding where you were working in the page. It's much different than the other comments throughout. Maybe have just one #